app/views/users/show.html.erb

```diff
@@ -15,6 +15,7 @@
           <th>Date</th>
           <th>Description</th>
           <th colspan="1">Actions</th>
+          <th> Delete </th>
```

Comment:
I think is better to increase the "Actions" `colspan` instead of creating new column

Reply:
this will change when the front is done.